lib/pubsub/index.js

```diff
@@ -160,8 +160,16 @@ PubSub.prototype.getTopics = function(query, callback) {
  *
  * @example
  * pubsub.createTopic('my-new-topic', function(err, topic, apiResponse) {
- *   topic.publish('New message!', function(err) {});
+ *   topic.publish({
```

Comment:
I don't think we have that yet: https://github.com/GoogleCloudPlatform/gcloud-node/issues/629